homeassistant/components/repairs/issue_handler.py

```diff
@@ -37,7 +36,13 @@ async def async_step_confirm(
         if user_input is not None:
             return self.async_create_entry(title="", data={})
 
-        return self.async_show_form(step_id="confirm", data_schema=vol.Schema({}))
+        issue_registry = async_get_issue_registry(self.hass)
+        issue = issue_registry.async_get_issue(self.handler, self.issue_id)
+
+        kwargs = {"step_id": "confirm", "data_schema": vol.Schema({})}
+        if issue:
+            kwargs["description_placeholders"] = issue.translation_placeholders
+        return self.async_show_form(**kwargs)
```

Comment:
I'm not a fan of kwargs dictionaries (hurts typing). Alternatively, this works exactly the same and retains the arguments in the `async_show_form` call.

```suggestion
        issue_registry = async_get_issue_registry(self.hass)
        description_placeholders = None
        if issue := issue_registry.async_get_issue(self.handler, self.issue_id):
            description_placeholders = issue.translation_placeholders

        return self.async_show_form(
            step_id="confirm",
            data_schema=vol.Schema({}),
            description_placeholders=description_placeholders,
        )
```